go.mod

```diff
@@ -9,10 +9,10 @@ require (
 	github.com/onsi/gomega v1.27.5
 	github.com/pkg/errors v0.9.1
 	gopkg.in/yaml.v2 v2.4.0
-	helm.sh/helm/v3 v3.10.0
+	helm.sh/helm/v3 v3.11.0
```

Comment:
how did you end up fixing the issues with k8s api and controller runtime getting bumped too?

Reply:
I copied the versions we were using in CAPZ. I believe we just needed a combination of versions that were mutually compatible, and since CAPZ compiles with those versions, it should work here too.

Reply:
Does anything change when you run `go mod tidy`?

Reply:
Nope